webapp/channels/src/components/profile_popover/components/profile_popover_title.tsx

```diff
@@ -121,24 +93,39 @@ const ProfilePopoverTitle = ({
             />
         );
     }
-    const titleClassName = classNames('popover-title', {'popover-title_height': !roleTitle});
 
     return (
-        <div className={titleClassName}>
-            <span data-testid={`profilePopoverTitle_${username}`}>
-                {roleTitle}
-                <button
-                    ref={closeButtonRef}
-                    className='user-popover__close'
-                    onClick={handleClose}
-                >
-                    <CloseIcon
-                        size={18}
-                    />
-                </button>
-            </span>
+        <div className={'user-profile-popover-title'}>
+            {roleTitle}
+            <button
+                className='btn btn-icon btn-sm'
+                onClick={handleClose}
+            >
+                <i className='icon icon-close'/>
+            </button>
         </div>
     );
 };
 
+function getIsTeamAdmin(state: GlobalState, userId: string) {
```

Comment:
Moving these selectors/helper functions below the component is the opposite of @larkox's preference of having them above the component. Personally, I'm also a fan of having them below, but since it makes little difference which side of the component they're on, I propose that we leave them wherever the original author decided to put them. That way, we won't make git blame harder to read and we won't have to spend review time going back and forth about the order of the file. How does that sound to both of you?

@M-ZubairAhmed That said, could you move these back in this case? I don't want to go back and forth on these, but since Daniel's the one who split these components out, I think we should stick with his order for them

Reply:
reverted